dev/com.ibm.ws.org.apache.cxf.cxf.core.3.2/bnd.bnd

```diff
@@ -8,11 +8,11 @@
 # Contributors:
 #     IBM Corporation - initial API and implementation
 #*******************************************************************************
--include= jar:${fileuri;${repo;org.apache.cxf:cxf-core;3.2.6}}!/META-INF/MANIFEST.MF,bnd.overrides
+-include= jar:${fileuri;${repo;org.apache.cxf:cxf-core;3.4.0}}!/META-INF/MANIFEST.MF,bnd.overrides
```

Comment:
we should always specify EXACT on the versions to avoid potential "soft matching" artifacts in Artifactory. I've had to sort out those issues before where a new artifact gets uploaded and for some reason it starts matching to the wrong version.